web/client/components/app/StandardApp.jsx

```diff
@@ -61,7 +64,9 @@ const StandardApp = React.createClass({
     },
     init() {
         this.store.dispatch(changeBrowserProperties(ConfigUtils.getBrowserProperties()));
-
+        if (urlQuery.appConfig) {
```

Comment:
appConfig.js is a name used to load the StandardApplication config, so it can confuse. Let's use localConfig.